doc/extensions.md

```diff
@@ -260,4 +260,36 @@ And with the listeners
     TimeZoneTestListener(TimeZone.getTimeZone(ZoneId.of("America/Sao_Paulo")))
   )
 
+```
+
+### Current instant listeners
+
+Sometimes you may want to user `java.time` classes' `now` static function for multiple reasons, such as setting the creation date of an entity
```

Comment:
Sometimes you may want to use the `now` static functions located in `java.time` ...

Reply:
much better. Brainfarted there